src/helpers/schema-miller.ts

```diff
@@ -59,53 +56,37 @@ export default class SchemaMiller {
 
     // Traverse the schema and collect generated files
     if (node.mill) {
-      const normal = SchemaMiller.normalizeOptions(node.opts || {})
+      const normal = SchemaMiller.normalizeOptions(node)
       const resolved = SchemaMiller.resolveDependency(normal, dir)
-      let headers = {}
       let form
-      if (resolved.opts.use) {
+      if (resolved.use) {
         form = undefined
       } else if (typeof payload === 'function') {
         form = payload()
       } else {
         form = payload
       }
-      if (form && form.getHeaders) {
-        headers = form.getHeaders()
-      } else {
-        headers = {
-          'content-type': 'application/json'
```

Comment:
@andrewxhill do you remember why this was added? From my tests, it appears to no longer be needed due to fetch now handling this for us, but can you please confirm that this is indeed the case? Or point me to where/why this was needed?

Reply:
the api request with raw json were failing https://github.com/textileio/js-http-client/pull/59/files

Reply:
Ok, cool. I think we're good then, shouldn't need to supply this directly anymore, fetch should take care of that for us.